### PR TITLE
[zh] Fix minikube install note

### LINKED ISSUE #21529: Note shortcode with html comment wont be parsed correctly
**Note shortcode with html comment wont be parsed correctly**

**Problem:**

After `layouts/shortcodes/note.html` has been updated since [Change notes templating to be able to comply with Hugo 0.70. ](https://github.com/kubernetes/website/pull/20946), the HTML comment inside `{{< node >}}` will result in markdownify not working correctly.

> English version is ok.

English version docs: [confirm-installation](https://kubernetes.io/docs/tasks/tools/install-minikube/#confirm-installation)

en

Chinese version docs: [confirm-installation](https://kubernetes.io/zh/docs/tasks/tools/install-minikube/#%E5%AE%89%E8%A3%85%E7%A1%AE%E8%AE%A4)

zh

**Proposed Solution:**

Move the HTML comment outside the `{{< node >}}` shortcode, it does work correctly. Just like this.

```
<!--
For setting the `--vm-driver` with `minikube start`, enter the name of the hypervisor you installed in lowercase letters where `<driver_name>` is mentioned below. A full list of `--vm-driver` values is available in [specifying the VM driver documentation](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).
-->

{{< note >}}

通过 `minikube start` 设置 `--vm-driver`。在下面提到 `<driver_name>` 的地方，用小写字母，输入你安装的 hypervisor 的名称。
[指定 VM 驱动程序](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver) 列举了 `--vm-driver` 值的完整列表

{{< /note >}}
```

### PR DESCRIPTION
**Pull request by tengqm**
fixes #21529

### CODE REVIEW

**Comment by tengqm** [comment]
Great. Moving the HTML comment out of the note block solved the problem.
Preview: https://deploy-preview-21706--kubernetes-io-master-staging.netlify.app/zh/docs/tasks/tools/install-minikube/

File: content/zh/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -393,14 +393,13 @@ To confirm successful installation of both a hypervisor and Minikube, you can ru
...
-{{< note >}}
 <!--
 For setting the `--vm-driver` with `minikube start`, enter the name of the hypervisor you installed in lowercase letters where `<driver_name>` is mentioned below. A full list of `--vm-driver` values is available in [specifying the VM driver documentation](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver).
 -->
 
-通过 `minikube start` 设置 `--vm-driver`。在下面提到 `<driver_name>` 的地方，用小写字母，输入你安装的 hypervisor 的名称。
-[指定 VM 驱动程序](https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-vm-driver) 列举了 `--vm-driver` 值的完整列表
-
+{{< note >}}
+若要为 `minikube start` 设置 `--vm-driver`，在下面提到 `<driver_name>` 的地方，用小写字母输入你安装的 hypervisor 的名称。
```
Comment (RA489) [inline content/zh/docs/tasks/tools/install-minikube.md:401]: [1] flag` --vm-driver has been depcrecated.` Do you mind updating it with` --driver` flag.

File: content/zh/docs/tasks/tools/install-minikube.md [hunk 2]
```diff
@@ -393,14 +393,13 @@ To confirm successful installation of both a hypervisor and Minikube, you can ru
...
+[指定 VM 驱动程序](/docs/setup/learning-environment/minikube/#specifying-the-vm-driver) 列举了 `--vm-driver` 值的完整列表。
```
Comment (RA489) [inline content/zh/docs/tasks/tools/install-minikube.md:402]: [1]

File: content/zh/docs/tasks/tools/install-minikube.md (hunk 1 above)
Comment (tengqm) [reply content/zh/docs/tasks/tools/install-minikube.md:401]: yup, but I think that belongs to a different PR. We need a separate PR to do a thorough update of the deprecated command line flag.

Comment (RA489) [reply content/zh/docs/tasks/tools/install-minikube.md:401]: +1

**Pull request merged**